apps/settings/const.py

```diff
@@ -5,3 +5,21 @@ class ImportStatus(TextChoices):
     ok = 'ok', 'Ok'
     pending = 'pending', 'Pending'
     error = 'error', 'Error'
+
+
+class ChatAITypeChoices(TextChoices):
+    gpt = 'gpt', 'GPT'
+    deep_seek = 'deep-seek', 'DeepSeek'
+
+
+class GPTModelChoices(TextChoices):
+    gpt_4o_mini = 'gpt-4o-mini', 'gpt-4o-mini'
+    gpt_4o = 'gpt-4o', 'gpt-4o'
+    o3_mini = 'o3-mini', 'o3-mini'
+    o1_mini = 'o1-mini', 'o1-mini'
+    o1 = 'o1', 'o1'
+
+
+class DeepSeekModelChoices(TextChoices):
+    deepseek_chat = 'deepseek-chat', 'DeepSeek-V3'
+    deepseek_reasoner = 'deepseek-reasoner', 'DeepSeek-R1'
```

Comment:
The code does not appear to contain any obvious irregularities or potential issues at this time.

Regarding optimizations:

- To optimize function call performance, you could make use of generators instead of creating lists when looping through `model_options`.

However, with respect to your specific request of checking only for minor changes over the last few years compared to September 2021, I would need additional context about what is different between the provided source and where it was written (to which year). Please note that while my responses are typically very concise, they may not cover everything mentioned here in full detail without additional constraints or specifics on the content being analyzed.